integration-tests/e2e/emaildomain.create.cy.js

```diff
@@ -0,0 +1,81 @@
+const MenuPage = require('../pages/menuPage')
+const EmailDomainListingPage = require('../pages/emailDomainListingPage')
+const EmailDomainCreationPage = require('../pages/emailDomainCreationPage')
+
+context('CreateEmailDomain', () => {
+  before(() => {
+    cy.clearCookies()
+  })
+
+  beforeEach(() => {
+    cy.task('reset')
+  })
+
+  it('Should create new email domain, after completing form and clicking the  create email domain button', () => {
+    cy.task('stubSignIn', { roles: [{ roleCode: 'MAINTAIN_EMAIL_DOMAINS' }] })
+    cy.signIn()
+    cy.task('stubAllEmailDomains', {})
+    MenuPage.verifyOnPage().viewEmailDomainListing()
+    const emailDomainListingPage = EmailDomainListingPage.verifyOnPage()
+    emailDomainListingPage.navigateToCreateEmailDomainPageButton().click()
+    const emailDomainCreationPage = EmailDomainCreationPage.verifyOnPage()
+    emailDomainCreationPage.createEmailDomainPage('Domain1', 'Domain1Description')
+    cy.task('verifyGetAllEmailDomains').should((requests) => {
+      expect(requests).to.have.lengthOf(1)
+    })
+  })
+
+  it('Should display error and prevent form submission, when email domain description is blank', () => {
+    cy.task('stubSignIn', { roles: [{ roleCode: 'MAINTAIN_EMAIL_DOMAINS' }] })
+    cy.signIn()
+    cy.task('stubAllEmailDomains', {})
+    MenuPage.verifyOnPage().viewEmailDomainListing()
+    const emailDomainListingPage = EmailDomainListingPage.verifyOnPage()
+    emailDomainListingPage.navigateToCreateEmailDomainPageButton().click()
```

Comment:
Probably easier/neater to just have a method that gets you to the create email domain page - then all the tests can use it

Reply:
Probably, yes.